packages/preferences/src/browser/views/components/preference-node-renderer.ts

```diff
@@ -170,6 +175,65 @@ export abstract class PreferenceLeafNodeRenderer<ValueType extends JSONValue, In
         this.inspection = this.preferenceService.inspect<ValueType>(this.id, this.scopeTracker.currentScope.uri);
     }
 
+    protected buildMarkdownRenderer(): MarkdownRenderer {
+        return new MarkdownRenderer()
+            .modify('a', link => {
+                const href = link.href;
+                link.href = '#';
+                link.title = href;
+                link.addEventListener('click', e => this.openLink(e, href));
+            })
+            .modify('code', code => {
+                const innerText = code.innerText;
+                // Linked preferences always start and end with `#`
+                if (innerText.startsWith('#') && innerText.endsWith('#')) {
+                    const preferenceId = innerText.substring(1, innerText.length - 1);
+                    const preferenceNode = this.model.getNodeFromPreferenceId(preferenceId);
+                    if (preferenceNode) {
+                        let name = this.labelProvider.getName(preferenceNode);
+                        const prefix = this.labelProvider.getPrefix(preferenceNode);
+                        if (prefix) {
+                            name = prefix + name;
+                        }
+                        const link = document.createElement('a');
+                        link.addEventListener('click', e => this.selectPreference(e, preferenceId));
+                        link.innerText = name;
+                        link.title = `#${preferenceId}`;
+                        link.href = '#';
+                        return link;
+                    }
+                }
+                return code;
+            });
+    }
+
+    protected openLink(event: MouseEvent, href: string): void {
+        event.preventDefault();
+        event.stopPropagation();
+        // Exclude right click
+        if (event.which < 3) {
```

Comment:
looks like `event.button` is the preferred way to figure out what button was clicked. Is there a reason not to use it here? (Same at l. 223)

Reply:
I got the inspiration from here: https://github.com/microsoft/vscode/blob/7be48d527a1a7fdf5fbaf4fc7af52195e6fa0bac/src/vs/code/electron-sandbox/issue/issueReporterMain.ts#L1190-L1197

However, `event.button` seems to be the better solution 👍